2023/03/identifying-vulnerabilities-using-horusec.md

```diff
@@ -0,0 +1,112 @@
+---
+author: "Indra Pranesh Palanisamy, Kürşat Kutlu Aydemir and Edgar Mlowe"
```

Comment:
Our blog only allows for one author, so please just choose the primary author here. If you want to give credit to others, you can mention them in a note at the end.

Reply:
I understand, I have added a short note at the end to give credits.